<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>10</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.971591, y2:0.949, stop:0 rgba(0, 0, 180, 255), stop:0.994318 rgba(164, 0, 171, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-30</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.0965909 rgba(0, 28, 70, 255), stop:0.914773 rgba(176, 39, 255, 255))</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="home">
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.0965909 rgba(0, 28, 70, 255), stop:0.914773 rgba(176, 39, 255, 255))</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>401</width>
        <height>371</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Добро пожаловать в нашу программу о туристических маршрутах! Здесь вы найдете множество увлекательных и захватывающих маршрутов для путешествий по Москве в пределах Садового кольца. Готовы отправиться в увлекательное путешествие? Тогда давайте начнем!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;добавить количество избранных в маршрутах&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="shapka">
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.0965909 rgba(0, 28, 70, 255), stop:0.914773 rgba(176, 39, 255, 255))</string>
     </property>
     <widget class="QPushButton" name="button_home12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>вкладки/back_icon-icons.com_65134.png</normaloff>вкладки/back_icon-icons.com_65134.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>411</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane
{
border:0
}
QTabBar::tab
	{
	
	background-color: rgb(184, 70, 255);
}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="marshutu">
       <attribute name="title">
        <string>Все</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="moee">
       <attribute name="title">
        <string>Избранное</string>
       </attribute>
      </widget>
     </widget>
     <zorder>tabWidget</zorder>
     <zorder>button_home12</zorder>
    </widget>
    <widget class="QWidget" name="karta">
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.0965909 rgba(0, 28, 70, 255), stop:0.914773 rgba(176, 39, 255, 255))</string>
     </property>
     <widget class="QPushButton" name="button_home_1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>вкладки/back_icon-icons.com_65134.png</normaloff>вкладки/back_icon-icons.com_65134.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Hello world</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="poisk">
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.0965909 rgba(0, 28, 70, 255), stop:0.914773 rgba(176, 39, 255, 255))</string>
     </property>
     <widget class="QPushButton" name="button_home_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>вкладки/back_icon-icons.com_65134.png</normaloff>вкладки/back_icon-icons.com_65134.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.0965909 rgba(0, 28, 70, 255), stop:0.914773 rgba(176, 39, 255, 255));
border-radius: 20px;  </string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Naity">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border-radius: 20px;  </string>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>371</width>
        <height>311</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
     <zorder>listView</zorder>
     <zorder>lineEdit</zorder>
     <zorder>Naity</zorder>
     <zorder>button_home_2</zorder>
    </widget>
    <widget class="QWidget" name="settings_2">
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.0965909 rgba(0, 28, 70, 255), stop:0.914773 rgba(176, 39, 255, 255))</string>
     </property>
     <widget class="QPushButton" name="button_home_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>вкладки/back_icon-icons.com_65134.png</normaloff>вкладки/back_icon-icons.com_65134.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ocenla">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>320</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;  </string>
      </property>
      <property name="text">
       <string>Оцените наш проект</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border-radius: 20px;  </string>
      </property>
      <property name="text">
       <string notr="true">Сменить тему</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="lupabutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/kandinsky-download-1701422191040.png</normaloff>../../Downloads/kandinsky-download-1701422191040.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kartatabutton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>вкладки/kandinsky-download-1701422488749.png</normaloff>вкладки/kandinsky-download-1701422488749.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shapkakabutton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>вкладки/kandinsky-download-1701422687778.png</normaloff>вкладки/kandinsky-download-1701422687778.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsbutton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>вкладки/kandinsky-download-1701422870435.png</normaloff>вкладки/kandinsky-download-1701422870435.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>lupabutton</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>kartatabutton</zorder>
   <zorder>shapkakabutton</zorder>
   <zorder>settingsbutton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
